data/brands/shop/supermarket.json

```diff
@@ -7746,6 +7746,17 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "Supermarket (Brasil)",
```

Comment:
The disambiguating suffix might not be necessary, since we don’t have any other preset named “Supermarket”.

So far, I think all the other disambiguators have been in English (“Brazil”), but only because we don’t have a solution for localizing presets by language, only by geography. Fortunately, this preset is somewhat unlikely to be shown to users of languages other than Portuguese.